client/verta/verta/_registry/model.py

```diff
@@ -15,6 +15,20 @@
 
 
 class RegisteredModel(_ModelDBEntity):
+    """
+    Object representing a registered model.
+
+    There should not be a need to instantiate this class directly; please use
+    :meth:`client.set_registered_model`.
```

Comment:
I think this will enable the generated doc to have a link to the Client method. Would you be able to build the docs locally and verify this?
```suggestion
    :meth:`Client.set_registered_model() <verta.client.Client.set_registered_model>`
```

Reply:
Not yet. I'll verify this and let you know later.

Reply:
Yep works like a charm.